language/transaction-builder/generator/tests/generation.rs

```diff
@@ -298,6 +297,181 @@ fn test_that_java_code_compiles_and_demo_runs() {
     assert!(output.status.success());
 }
 
+#[test]
```

Comment:
This test should probably be ignored by default. But we'll make sure to run it in CI

Reply:
https://github.com/diem/diem/pull/7816